### PR TITLE
Small sized icons in settings

### PR DESCRIPTION
**Pull request by signedav**
Made an own Switch.qml overwriting the Controls.Switch to set the sizes depending on the dpi.
On the Galaxy S6 mobile phone:

screenshot_20180827-154135

Fix of: #333

### CODE REVIEW

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://opengis.ch/download/qfield/ci-builds/qfield-dev-337-9a197b3bf79349e150729843722913b3e083cfb2-armv7.apk) and [x86](https://opengis.ch/download/qfield/ci-builds/qfield-dev-337-9a197b3bf79349e150729843722913b3e083cfb2-x86.apk)

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://opengis.ch/download/qfield/ci-builds/qfield-dev-337-7f06b606ee1a691b5d4907ee4b39017342d8d37f-armv7.apk) and [x86](https://opengis.ch/download/qfield/ci-builds/qfield-dev-337-7f06b606ee1a691b5d4907ee4b39017342d8d37f-x86.apk)

**Comment by VxTedxV** [comment]
related to #333

File: src/qml/QFieldSettings.qml [hunk 1]
```diff
@@ -7,6 +7,8 @@ import QtQuick.Layouts 1.3
 
 import "js/style.js" as Style
 
+import "."
```
Comment (m-kuhn) [inline src/qml/QFieldSettings.qml]: What is that for?

Comment (m-kuhn) [reply src/qml/QFieldSettings.qml]: Oh, I see

**Pull request merged**